types/pixi/grids/GridHighlight.d.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * A special Graphics class which handles Grid layer highlighting
+ */
+declare class GridHighlight extends PIXI.Graphics {
+  /**
```

Comment:
We changed the conventions a bit since you last posted. Constructors now go first in the class.
```suggestion
declare class GridHighlight extends PIXI.Graphics {
  constructor(name: any, ...args: any[])
  
  /**
```